web/packages/teleport/src/components/Layout/Layout.tsx

```diff
@@ -17,15 +17,14 @@
  */
 
 import styled from 'styled-components';
-import { Flex, Text } from 'design';
+import { Flex, H1 } from 'design';
 import { alignItems, space } from 'design/system';
 
 /**
  * Header
  */
 const FeatureHeader = styled(Flex)`
```

Comment:
I'm not sure if this should be addressed in this PR, but spacing around `FeatureHeader` doesn't look good.
Now:
<img width="1260" alt="image" src="https://github.com/user-attachments/assets/ac692db3-a13c-46f3-b6e5-28d5235ce901">

With `margin-top` and `margin-bottom` = 12px:
<img width="1260" alt="image" src="https://github.com/user-attachments/assets/3b3d1b3b-8fff-41f5-afc5-db06fa94d10c">

Reply:
Indeed, it looks better this way, but outside of the scope.